openedx_caliper_tracking/tasks.py

```diff
@@ -147,7 +147,8 @@ def sent_kafka_failure_email(self, error):
 
     data = {
         'name': 'UCSD Support',
-        'body': 'Below is the additional information regarding failure:',
+        'body': 'Below is the additional information regarding failure:\n LMS_ROOT_URL = {}'.format(
```

Comment:
lets use something other that `LMS_ROOT_URL` in the PR. it's not user friendly.
We can using something like `System URL =` or something like that? Can you suggest something better?